### ISSUE TITLE
Command Table Helper - Passing integers to cells

### ISSUE THREAD

**Issue by mentisy** [cakephp/cakephp#15327]
This is a (multiple allowed):

* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* CakePHP Version: EXACT RELEASE VERSION OR COMMIT HASH, HERE.

### What you did
I have a console command for viewing members in my project. In the command I use the built-in table helper to output the entities with their properties and values. The entity contains integers, as well as strings.
```php
public function execute(Arguments $args, ConsoleIo $io): ?int
{
    $members = [
        ['Id', 'Name', 'Age'],
        [1, 'Jane', 27],
        [2, 'John', 25],
    ];

    $this->io->helper('Table')->output($members);
}
```

### What happened
I get an exception saying the argument passed to `TableHelper::_cellWidth()` must be of type string or null.
```
2021-02-11 16:14:39 Error: [TypeError] Argument 1 passed to Cake\Shell\Helper\TableHelper::_cellWidth() must be of the type string or null, int given, 
called in /vendor/cakephp/cakephp/src/Shell/Helper/TableHelper.php on line 49 in /vendor/cakephp/cakephp/src/Shell/Helper/TableHelper.php on line 65
Stack Trace:
- /vendor/cakephp/cakephp/src/Shell/Helper/TableHelper.php:49
- /vendor/cakephp/cakephp/src/Shell/Helper/TableHelper.php:150
- /src/Command/EditMemberCommand.php:68
- /vendor/cakephp/cakephp/src/Console/BaseCommand.php:179
- /vendor/cakephp/cakephp/src/Console/CommandRunner.php:336
- /vendor/cakephp/cakephp/src/Console/CommandRunner.php:172
- /bin/cake.php:12
```

### What you expected to happen
I expected the helper to either accept integers or cast them into strings.

I'm unsure whether you guys qualify this as unexpected behavior. Sure, I could cast them manually myself before passing them to the helper, but that shouldn't be necessary *in my opinion*.

I believe passing integers to the table worked fine until this commit change: https://github.com/cakephp/cakephp/commit/9fc0363079d08f8ecefd5394c919ae5edd6951b2

**Comment by mentisy** [comment]
I made a PR (see #15328) if you would like to allow passing integers.

**Comment by mentisy** [comment]
Solved by #15328
